coalib/bearlib/aspects/decorators.py

```diff
@@ -49,3 +49,51 @@ def _new_func(self, *args, **kwargs):
         return _new_func
 
     return _func_decorator
+
+
+def map_ambiguous_setting_to_aspect(**aspectable_setting):
+    """
+    Map those bear settings to aspects in which there is some ambiguity.
+    The ambiguity is because of differing values (maybe either name or even
+    data-types) in bear settings and their corresponding aspect tastes.
+
+    This decorator can be used by ``Bear.run()`` to automatically map and
+    override value of bear settings with their equivalent taste.
+
+    The order of setting override from the lowest to highest is:
+    - Setting default (in bear's run argument)
+    - Aspect/taste default (if aspect is activated in Section)
+    - Explicit aspect/taste default (if aspect is activated in Section)
+    - Explicit setting
```

Comment:
The order of setting override would be clearer if the order is reversed to :
The order of setting override from the highest to lowest is:

Explicit setting
Explicit aspect/taste default (if an aspect is activated in Section)
Aspect/taste default (if an aspect is activated in Section)
Setting default (in bear's run argument)